src/Card.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+import Box from './Box'
+import PropTypes from 'prop-types'
+
+function boxShadow(props) {
+
+  const boxShadows = {
+    'sm': {
+      'box-shadow': props.theme.boxShadows[0]
+    },
+    'md': {
+      'box-shadow': props.theme.boxShadows.slice(0,2).join(', ')
+    },
+    'lg': {
+      'box-shadow': props.theme.boxShadows.slice(0,3).join(', ')
+    },
+    'xl': {
+      'box-shadow': props.theme.boxShadows.join(', ')
+    }
+  }
+  return boxShadows[props.boxShadowSize]
+}
+
+const Card = styled(Box)`
+  ${boxShadow}
+`
+
+Card.propTypes = {
+  boxShadowSize: PropTypes.oneOf([
```

Comment:
Might be worth setting a defaultProp for whichever size we think is the most commonly used - cc @odmln